lesson18/main.js

```diff
@@ -94,22 +92,35 @@ const createBackBtn = () => {
   backBtn.addEventListener(
     "click",
     () => {
-      switchImg("previousElementSibling");
-      switchDisableForBtn();
-      getCurrentPageNum();
+      clickBtnEvents("previousElementSibling");
     },
     false
   );
 };
 
-const  switchImg = (SwitchDirection) => {
+const clickBtnEvents = (SwitchDirection) => {
```

Comment:
```suggestion
const clickBtnEvents = (switchDirection) => {
```

変数名の最初が大文字になる場合は固定値の時ぐらいです

Reply:
I didn't know this rules and now I've learned! Thank you for teaching me.
[ebabfe3](https://github.com/yuka830/js-lessons/pull/23/commits/ebabfe37a78508f2ff209a64567e893ffa247a57)